README.md

```diff
@@ -18,24 +18,24 @@ provides an accurate native V8 C++ API implementation available from PHP.
  - provides up-to-date JavaScript engine with recent [ECMA](http://kangax.github.io/compat-table) features supported;
  - accurate native V8 C++ API implementation available from PHP;
  - solid experience between native V8 C++ API and V8 API in PHP;
- - no magic; no assumption;
- - does what it asked to do;
- - hides complexity with isolates and contexts scope management under the hood;
+ - no magic; no assumptions;
+ - does what it is asked to do;
+ - hides complexity by isolation and contexts scope management under the hood;
```

Comment:
There are `v8::Isolate` and `v8::Context` concepts in v8 engine. Isolate could be accessed by only one thread at a time and multiple Contexts could reside in it at the same time. To perform some operations in v8 in C++ you need to enter Isolate, enter Context, do you job and then leave it (it's not pretty accurate explanation, but for simplicity let it be this way at this time).
 
So I tried to say that this entering/leaving is hidden under the hood and done automatically when needed transparently to the user.